jenkins/job-configs/kubernetes-jenkins/clean-ci-node-e2e.yaml

```diff
@@ -0,0 +1,18 @@
+- job:
+    name: 'clean-ci-node-e2e'
+    description: 'Clean up leaked node e2e resources. Test owner: test-infra maintainers.'
+    node: ''
+    triggers:
+        - timed: 'H H/3 * * *'
+    properties:
+        - build-discarder:
+            days-to-keep: 14
+    wrappers:
+        - e2e-credentials-binding
```

Comment:
How come? We don't yet have an alternative to this, and we need it in order to upload the build log at the end of the run.

Every job will now activate the service account using these credentials, see:

https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L507
and https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L506